model/workflow.go

```diff
@@ -206,6 +206,14 @@ func (w *Workflow) setDefaults() {
 	}
 }
 
+// WorkflowRef holds a reference for a workflow definition
+type WorkflowRef struct {
```

Comment:
This struct is used by Action & Workflow struct, so should we place it in workflow_ref.go file?

Reply:
it will have just one thing in there, maybe later as it grows?

Reply:
maybe after we get your PR merged.